### DISCUSSION TITLE
About FastMRI ISMRM header file

### DISCUSSION THREAD

**Discussion by mclvnot** [facebookresearch/fastMRI#312] [Q&A]
Hello,

I want to see scans multiplanar version of scans and for conversion I need to have ImagePatientPosition and ImagePatientOrientation tag. But in measurements from header, I could not have these tags. Is it possible to have or is it released with meta file?

Best,

**Comment by mmuckley** [comment]
Hello @mclvnot, as you can see in my comment on #313, you can get patient position from the header under `measurementInformation`. I don't see a formal patient orientation in the header. There is the protocol name that specifies a coronal scan, but this is entered by the tech and may not be consistent across all scans. This would be the only information released.

**Comment by mclvnot** [comment]
It is not position it is attribute about  patient position named FFS. Image patient position is worl coordinate with mm(x y z ),  for coronal orientation I used orientation [1 0 0 0, 0 0 0 -1] but this is general orientation not patient dependent. 

I dont have no idea what I have to do for calculaton patient position and orientation.

Best,

**Reply by mmuckley** [reply]
Hello @mclvnot are you talking about the DICOM field `ImagePatientPosition`? In the future I recommend specifying which standard you are referring to. There are dozens of standards for medical imaging data, often with overlapping names.

That being said, I don't think you can get this information. Why would you need it? Are you trying to co-register the reconstructions with something else?

**Reply by mmuckley** [reply]
If you just want to view multiplanar scans you can just define all these attributes manually.

**Comment by mclvnot** [comment]
Yes I mean DICOM field, I converted hdf5 to dicom but for multiplanar reconstruction, I need to have ImagePatientPosition, ImagePatientOrientation and FrameUID tags. I could get FrameUID from hdf5 ISMRM header file but other tags are not existent. 

I did some parameter changing in tags, but not calculated for axial and saggital plane due to random coordinates. Nobody knows how to calculate position and orientation with pixel spacing and slice thickness around me and says it must be taken from scanner.

Best,

**Reply by mmuckley** [reply]
Okay, we might have to wait for some others to comment. We were actually able to do most of our analysis without writing the 3D coordinates and generally avoided touching the DICOM standard too much. For our more clinically-oriented work, we usually have a dedicated, non-anonymized dataset with full DICOM tags.

Sorry, not sure I can solve everything you're looking for.
